weather_app/models.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import unicode_literals
+from django.db import models
+import uuid
+
+""" Custiom time stamp model """
+class CustomTimeStampModel(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    modified_at = models.DateTimeField(auto_now=True)
+    is_active = models.BooleanField(default=True)
+
+    """ using to set class abstract for to create a model """
+    class Meta:
+        abstract = True
+
+""" Weather Characteristics model """
+class WeatherCharacteristic(CustomTimeStampModel):
+    min_value = models.DecimalField(max_digits=5, decimal_places=1)
+    max_value = models.DecimalField(max_digits=5, decimal_places=1)
+    avg_value = models.DecimalField(max_digits=5, decimal_places=1)
+
+    """ return back fields to the admin panel """
+    def __str__(self):
+        return r"{0} - {1} - {2}".format(self.min_value,self.max_value,self.avg_value)
```

Comment:
use [f-string](https://www.python.org/dev/peps/pep-0498/) for string representation.